soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -1594,9 +1408,14 @@ s32 func_808332E4(Player* this) {
 }
 
 void func_808332F4(Player* this, GlobalContext* globalCtx) {
-    GetItemEntry* giEntry = &sGetItemTable[this->getItemId - 1];
+    GetItemEntry giEntry;
+    if (this->getItemEntry.objectId == OBJECT_INVALID) {
+        giEntry = ItemTable_Retrieve(this->getItemId - 1);
```

Comment:
would removing the `-1` here make it so less counteracting would be needed?

Reply:
As mentioned in an earlier comment, the Vanilla GetItemTable's indices are off by one from GetItemId values. I can probably change that now that the Vanilla GetItemTable is a map though. Was trying not to change vanilla code too much if possible but that's probably what needs to happen.

Reply:
Addressed by 69086f58c2b6bb00a273eefed4744ad456bba6ba